Components/WhiteBox.js

```diff
@@ -0,0 +1,20 @@
+import React, { PropTypes } from 'react';
+
+import '../css/Components/WhiteBox';
+
+import Mirrored from './Mirrored';
+
+const WhiteBox = ({ children, id }) => (
+  <Mirrored id={id} hide>
```

Comment:
I would prefer if you moved this into a higher level component, but if you fell/know that all the white boxes will be hidden from the mirror effect leave it as it is.